### PR TITLE
fix response with list object

### PR DESCRIPTION
**Pull request by nllong**
This fixes cleansing. It broke after we tried to standardize the return codes. Please review and make sure my change is appropriate.

### CODE REVIEW

**Comment by rebrownlbnl** [comment]
Is it possible to add a test to detect the original problem?

File: seed/decorators.py [hunk 1]
```diff
@@ -99,8 +99,12 @@ def wrapper(request, *args, **kwargs):
         response = func(request, *args, **kwargs)
 
         status_code = 200
-        if response.get('status') == 'error' or response.get('success') == False:
-            status_code = 400
+
+        # This is somewhat hackish, but the response is not always an HttpResponse. I'm explicitly
+        # excluding the list because it is a known type that is returned in the cleansing routines.
+        if not isinstance(response, list):
```
Comment (mmclark) [inline seed/decorators.py]: Would it make more sense to test if response is an HttpResponse before calling get(), as opposed to testing if it's not a list?

**Comment by nllong** [comment]
This will fix #459

File: seed/decorators.py (hunk 1 above)
Comment (nllong) [inline seed/decorators.py]: @mmclark -- i'm going to fix this today and add a test.

**Pull request merged**